components/Player/PlayerRecord.jsx

```diff
@@ -3,17 +3,22 @@ import Error from '../Error';
 import Spinner from '../Spinner';
 import { connect } from 'react-redux';
 import { player } from '../../reducers';
-import styles from './PlayerHeader.css';
+// import styles from './PlayerHeader.css';
 
 export const PlayerRecord = ({ loading, error, wins, losses }) => {
-  const calcWinPercent = (wins, losses) => Math.ceil(10000 * ((wins) / (wins + losses))) / 100;
-
   const getPlayerRecord = () => {
     if (error) return <Error />;
     if (loading) return <Spinner />;
     return (
-      <div className={styles.mmr}>
-        <div>{`${wins} - ${losses} (${calcWinPercent(wins, losses)}%)`}</div>
+      <div>
+        <span>
+          <small>
+            <span>{wins}</span>
+            <span> - </span>
+            <span>{losses}</span>
+            <span>{` (${(wins / (wins + losses) * 100).toFixed(2)}%)`}</span>
```

Comment:
I don't think we need so many different span tags unless it's for line breaking reasons, but you can use css for that.